### PR TITLE
Add i18n master 20200811

### PR DESCRIPTION
**Pull request by luke2327**
### Sinopsis

I'm inspired by your Legion Solver.
I want to make Korean MapleStory users understand and use it.

So I added i18n.

### Feature: i18n

add: GMS
add: KMS

### ETC

remove Illegal white spaces.
In Korean MapleStory, there is a special job called [MapleStory M], which I added to i18n.

### P.S.
If this were to be merged, Language to be added later: JMS (Japanese)

### CODE REVIEW

File: src/board.js [hunk 1]
```diff
@@ -367,8 +366,8 @@ function resetDuringPause() {
     document.getElementById("clearBoard").disabled = false;
     document.getElementById("boardButton").innerText = "Start";
     document.getElementById("resetButton").style.visibility = 'hidden';
-    document.getElementById("iterations").innerText = "";
-    document.getElementById("time").innerText = "";
+    document.getElementById("iterations").style.display = 'none';
```
Comment (Xenogents) [inline src/board.js:369]:
```suggestion
    document.getElementById("iterations").style.visibility = 'hidden';
```

File: src/locales/ko.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    iterations: '반복횟수: ',
+    time: '소요시간: ',
+
+    bigClick: '지역선택',
+    liveSolve: '실시간 보기',
+    darkMode: '다크모드',
+
+    start: '시작',
+    pause: 'Pause',
+    continue: 'Continue',
```
Comment (Xenogents) [inline src/locales/ko.js:20]: Could you also translate these?

**Review by Xenogents — COMMENTED** [review]
Thanks for the contribution. I've made a few changes but overall looks really good, and I'm happy to help the KMS community! Could you just translate a couple extra words and I'll merge it.

File: src/locales/ko.js (hunk 2 above)
Comment (luke2327) [reply src/locales/ko.js:20]: I wrote down the English word that I often use in Korea. Should I apply it if you need it?

Comment (Xenogents) [reply src/locales/ko.js:20]: Oh alright that's fine, I'll merge then.

**Review by Xenogents — APPROVED** [review]

**Comment by luke2327** [comment]
Yes! I want to add JMS !

**Pull request merged**